### PR TITLE
Add test params and visualize code

### PR DESCRIPTION
**Pull request by iory**

### CODE REVIEW

**Comment by mqcmd196** [comment]
LGTM, thanks!

File: jsk_perception/test/test_remove_blurred_frames.test [hunk 1]
```diff
@@ -15,6 +15,8 @@
       timeout_1: 20
       topic_2: /head_camera/rgb/image_rect_color/laplacian_var
       timeout_2: 20
+      check_after_kill_node: true
```
Comment (mqcmd196) [inline jsk_perception/test/test_remove_blurred_frames.test:18]: @iory 
I'm sorry, what is this test for?

Comment (iory) [reply jsk_perception/test/test_remove_blurred_frames.test:18]: This test is for nodelet's destructor.
https://github.com/jsk-ros-pkg/jsk_recognition/pull/2681

Comment (mqcmd196) [reply jsk_perception/test/test_remove_blurred_frames.test:18]: I see. This test fails in my environtment. Could you check please?

Comment (iory) [reply jsk_perception/test/test_remove_blurred_frames.test:18]: https://github.com/mqcmd196/jsk_recognition/pull/3
Sorry. This PR fixes the test.

Comment (mqcmd196) [reply jsk_perception/test/test_remove_blurred_frames.test:18]: Thank you!!

**Pull request merged**